libs/vectorise/include/vectorise/fixed_point/fixed_point.hpp

```diff
@@ -706,13 +707,19 @@ inline std::ostream &operator<<(std::ostream &s, int128_t const &x)
   return s;
 }
 
+inline std::ostream &operator<<(std::ostream &s, uint128_t const &x)
+{
+  s << static_cast<uint64_t>(x >> 64) << "|" << static_cast<uint64_t>(x);
```

Comment:
is this operator for printing? if so, presumably we don't want to print number as 123|456 but as 123.456?
what am i missing?

Reply:
no, this is only for printing the uint128_t value, the container, used only for debugging purposes.